include/modelstate.h

```diff
@@ -93,33 +86,16 @@ class ModelState
 private:
   ModelState(std::string);
   static ModelState * modelState;
-  //double *qS, *dqS, *dxbS;
-  double vbS[6],*qjS,*qjDotS, gS[3];//;qjS[MEX_WBMODEL_MAX_NUM_JOINTS],qjDotS[MEX_WBMODEL_MAX_NUM_JOINTS],vbS[6];
-//   wbi::Frame rootS;
-  int robot_reference_frame_link;
-  //wbi::Frame H_rootLink_wrWorld;
-  //wbi::Frame H_referenceLink_wrWorld;
-   //wbi::Frame world_H_root;
-   wbi::Frame world_H_reference;
+  double vbS[6],*qjS,*qjDotS, gS[3];
 
   size_t numDof;
   std::string currentRobotName;
-  std::string robot_reference_frame_link_name;
 
   static wbi::iWholeBodyModel *robotWBIModel;
 
-  bool fixedLinkComputation;
-
   Eigen::Matrix4d H_w2b;
```

Comment:
Unused attribute. @naveenoid  consider adding the -Wall option to the compiler options in the cmake when compiling in Debug, so you can get warning like this. Check here https://github.com/robotology/codyco-modules/blob/f7dfddfa4299c4e605c7b533c3fe4f04389a4f8b/cmake/modules/CoDyCoOptions.cmake#L83 how we do in codyco-modules .